src/ddmd/cond.d

```diff
@@ -64,6 +67,227 @@ extern (C++) abstract class Condition : RootObject
 }
 
 /***********************************************************
+ * This class implements common functionality for StaticForeachDeclaration and StaticForeachStatement
+ */
+
+immutable StaticForeach_tupleFieldName = "tuple"; // used in lowering
```

Comment:
1. Yes, that's an accident.

2. It does, but `extern(C++)` classes cannot have static immutable fields. It seems I can work around this by marking the field `extern(D)`. I'll do that.